contracts/contracts/extensions/DeletableFacet.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+import {TokenOwnerFacet} from './TokenOwnerFacet.sol';
+import {RolesFacet} from './RolesFacet.sol';
+// solhint-disable-next-line max-line-length
+import {IHederaTokenService} from '@hashgraph/smart-contracts/contracts/system-contracts/hedera-token-service/IHederaTokenService.sol';
+import {IDeletable} from './Interfaces/IDeletable.sol';
+import {_DELETABLE_RESOLVER_KEY} from '../constants/resolverKeys.sol';
+
+contract DeletableFacet is IDeletable, TokenOwnerFacet, RolesFacet {
```

Comment:
This is incorrect. A facet never extends to other Facets:
`contract DeletableFacet is IDeletable, TokenOwnerStorageWrapper, RolesStorageWrapper {`